docs/api/zopeconfigure.rst

```diff
@@ -2,11 +2,3 @@
 =======================================
 
 .. automodule:: zope.configuration.zopeconfigure
-
-.. autointerface:: IZopeConfigure
-   :members:
-   :member-order: bysource
-
-.. autoclass:: ZopeConfigure
-   :members:
-   :member-order: bysource
```

Comment:
Why delete these here?  Seems unrelated to the PR.

Reply:
They were already included from the '.. automodule:: ' above and so were getting documented twice.